REST_API/README.md

```diff
@@ -0,0 +1,151 @@
+# Call Kobiton REST API to get session information
+
+This repository includes examples to demonstrate how to execute the script to get Kobiton session information and get session commands using NodeJS.
+## 1. Get session information
+Copy your username, API key, session ID and replace them in the sample script.
+```
+const username='YOUR_USERNAME';
+const apikey='YOUR_API_KEY';
+
+const session = [SESSIONID];
+```
+Once you have everything set up, you can run the script by running the following command.
```

Comment:
This line should only contains two commands ("node get_session_info.js" or "npm get_session_info"). The "npm install" line should be placed before this line, clearly explained as a way to install missing dependencies.